week-3/F-array-forEach/exercise.js

```diff
@@ -8,6 +8,18 @@
 */
 
 var arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+arr.forEach(function(e) {
+  if (e % 3 == 0 && 5 == 0) {
```

Comment:
avoid using single letter for arguments to functions. element or number is a better choice